cocoa/src/toga_cocoa/app.py

```diff
@@ -297,6 +303,17 @@ def _create_app_commands(self):
                 group=toga.Group.HELP,
             ),
         )
+        if macos_version_at_least(10, 12):
```

Comment:
Good thinking, but probably not necessary. As a general rule, we don't support OS versions that the vendor themselves doesn't support; 10.12 (Sierra) has been EOL for 4 years. 

Admittedly, there's a discrepancy with the main README, which says Yosemite is supported; however, the Briefcase support packages for packaged apps set a hard minimum of 11 (Big Sur), so this is as good an opportunity as any to update those docs and avoid the extra branch.

Reply:
Oh, yeah I was going off of the readme. Duly noted.